austation/code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -27,4 +27,20 @@
 	if(istype(J))
 		return J
 
-
+/obj/item/drone_shell
+	name = "hand-held drone shell"
+	desc = "a drone shell yet to be activated. use it in your hand to activate the drone shell."
+	w_class = WEIGHT_CLASS_NORMAL
+	icon = 'icons/mob/drone.dmi'
+	icon_state = "drone_maint_grey"
+
+/obj/item/drone_shell/attack_self(mob/user)
+	. = ..()
+	var/turf/T = get_turf(src)
+	if(!T)
+		user.show_message("<span class='warning'> There is no room to build \the [src].</span>")
+		return
+	user.visible_message("<span class='notice'> [user] begins to build \a [src].</span>", "<span class='notice'> You begin to build \the [src].</span>", "<span class='notice'> You can hear someone building \a [src].</span>", 7)
```

Comment:
why does it say "begins to build" if it's instantly built?

Reply:
Apologies, this code was pasted from my balloon PR from April.
I DID actually read over it and notice that mistake, but for reasons I have now forgotten I decided to not alter the text unprompted.

Reply:
fixed